sdks/python/apache_beam/coders/standard_coders.yaml

```diff
@@ -0,0 +1,39 @@
+coder:
+  urn: "beam:coders:bytes:0.1"
+nested: false
+examples:
+  "abc": abc
```

Comment:
Hmm... I'll experiment with that. Certainly I want to support having a single set of examples for both.